lua/entities/cfc_trampoline/init.lua

```diff
@@ -4,6 +4,7 @@ AddCSLuaFile( "cl_init.lua" )
 AddCSLuaFile( "shared.lua" )
 
 include( "shared.lua" )
+util.AddNetworkString( "CFC_BouncyCircle_PlayBounceSound" )

```

Comment:
This should only be run on server. If you don't have a server file, just wrap an `if SERVER` around it

Reply:
That is in a server only file